internal/modelschema/modelschema.go

```diff
@@ -4,7 +4,7 @@
 // you may not use this file except in compliance with the License.
 // You may obtain a copy of the License at
 //
-//     http://www.apache.org/licenses/LICENSE-2.0
+//	http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
we don't want to replace the space characters with a tab character here

This happens because Go thinks this is a package comment instead of a license header.

We should revert the tab and instead add an empty line between the package declaration and the copyright header.

Same in several other files :-)

Reply:
I guess the tab here is expected.
See this example https://github.com/apache/incubator-kie-kogito-serverless-operator/blob/main/utils/cluster.go#L10

But yes, header should have a blank line after.

Reply:
remove tab as suggested.

Reply:
Thank you @spolti -- it's possible, that over time some of the copyright headers got modified inadvertently on unrelated source code changes by running `go fmt` which then treated the copyright header as a package comment.

Originally the license header looked like this (with spaces, no tabs):
```go
// Copyright 2021 IBM Corporation
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//     http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.
```

Reply:
Thanks for adding the empty lines above the package declaration. I would only do that on files that you actually had to modify, not doing a full sweep.